### PR TITLE
Add recommender api token

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <fabric8.version>2.2.157</fabric8.version>
     <fabric8.console.version>2.2.199</fabric8.console.version>
     <fabric8.devops.version>2.2.331</fabric8.devops.version>
-    <fabric8-team-components.version>3.0.25</fabric8-team-components.version>
+    <fabric8-team-components.version>3.0.27</fabric8-team-components.version>
```
Comment (rawlingsj) [inline pom.xml:77]: FWIW this has already been done by the team pipeline, there's no conflict as they match but just wanted to let you know @aslakknutsen

**Comment by rawlingsj** [comment]
add to whitelist

**Comment by rawlingsj** [comment]
[merge]

File: pom.xml (hunk 1 above)
Comment (aslakknutsen) [reply pom.xml:77]: Yeah I was looking for the PR but must have missed it.. :)

Comment (rawlingsj) [reply pom.xml:77]: too slow ;)

**Pull request merged**